pylti/flask.py

```diff
@@ -7,7 +7,10 @@
 import logging
 import json
 
+
+ # pylint: disable=import-error
```

Comment:
I am not fully sure, but if you add a hint for pylint on its own line, it applies to the whole block. So it is better to add to the end of the specific line.
```
from flask import session, current_app, Flask  # pylint: disable=import-error
```